BedrockServer.cpp

```diff
@@ -2016,9 +2016,12 @@ void BedrockServer::onNodeLogin(SQLitePeer* peer)
 void BedrockServer::_acceptSockets() {
     // Try block because we sometimes catch `std::system_error` from in here (likely from the thread code) and we're
     // trying to diagnose exactly what's happening.
+    static uint64_t lastLogged = 0;
```

Comment:
won't this reset the variable to 0 every time `_acceptSockets` is called?

Reply:
I don't think so because it's set to `static`